### ISSUE TITLE
Set tintColor for Camera button issue in ios 9

### ISSUE THREAD

**Issue by intersilindia** [tdginternet/TGCameraViewController#54]
I just tested the code in ios 9 and found that the tint color which has been set to camera shot button doesnt appear 
It seems to be an ios 9 issue since the same code works on ios 8 and prior,not sure though but still I would like you to please help me figure out the exact reason

**Comment by miketraverso** [comment]
I've had a user also report this on an iPhone 6+ on iOS9 but unfortunately I cannot reproduce this on an iPhone 6+ with iOS9

**Comment by philokezzar** [comment]
I got the same issue in iP6 iOS 9

**Comment by miketraverso** [comment]
So figured this out. The XIBs buttons have the TGTintButton class. As a result the image is never set. I'm planning to create a pull request later today with a fix where the code will explicitly set the correct images. Once that code is applied there is no longer an issue with missing image assets on the buttons. The reason flash always appears because that's explicitly being set within the code.

**Comment by woobione** [comment]
I have the same issue. Looking forward to your pull @miketraverso :)

**Comment by miketraverso** [comment]
BUG FIX: Update type to on return type.	e98b3cd
Added outlets and changed types of UIButton in code but TGTintedButto… …	88f6278
Added outlets and changed types of UIButton in code but TGTintedButto… …	 2652bd9

**Comment by ADSolutions** [comment]
any news?

**Comment by miketraverso** [comment]
I created the pull request a while back. Haven't seen any action on that and can't create my own branch. The links to my checkins are in the comment above though.

**Comment by ADSolutions** [comment]
@miketraverso thank you! i'll try it now

**Comment by iHardikTrivedi** [comment]
Thank you very much . . . 

It's work fine . . .

**Comment by 278swati** [comment]
Thanks a ton... it helps a lot in my project as it won't works with device when i change color ... ur solution is really works as magic !

**Comment by alexandre-g** [comment]
An easier fix would be just removing the `__weak` from these lines in TGTintedButton.m updateTintIfNeeded: method.
```
UIImage * __weak backgroundImage = [[self backgroundImageForState:UIControlStateNormal] imageWithRenderingMode:renderingMode];
[self setBackgroundImage:backgroundImage forState:UIControlStateNormal];
        
UIImage * __weak image = [[self imageForState:UIControlStateNormal] imageWithRenderingMode:renderingMode];
[self setImage:image forState:UIControlStateNormal];
```

Anyone here know if that will lead to leaked UIImages?

**Comment by MattiaConfalonieri** [comment]
@alexandre-g are you sure your easy fix work?

**Comment by alexandre-g** [comment]
@MattiaConfalonieri works very well, in 2 production apps

**Comment by abbasmousavi** [comment]
It works in my production app too.

**Comment by macecchi** [comment]
Fixed in #78 and released in v2.2.7.
